toolkit/tools/imagecustomizerapi/systemconfig.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+// SystemConfig defines how each system present on the image is supposed to be configured.
+type SystemConfig struct {
```

Comment:
Is this just the initial iteration and the plan is to add the remaining attributes in subsequent PRs?

Reply:
Yes. I am intentionally not adding the APIs until they are implemented. This makes it very clear what APIs are usable and which aren't.

(I have seen other projects that preemptively added all the APIs, only for a ton of them to remain unimplemented indefinitely.)